internal/tools/pipeline/providers/cron/db/define.go

```diff
@@ -35,9 +35,9 @@ const (
 
 type PipelineCron struct {
 	ID            uint64    `json:"id" xorm:"pk autoincr"`
-	TimeCreated   time.Time `json:"timeCreated" xorm:"created"`   // 记录创建时间
-	TimeUpdated   time.Time `json:"timeUpdated" xorm:"updated"`   // 记录更新时间
-	SoftDeletedAt uint64    `json:"softDeletedAt" xorm:"deleted"` // 记录删除时间(时间戳形式)
+	TimeCreated   time.Time `json:"timeCreated" xorm:"created"`           // 记录创建时间
+	TimeUpdated   time.Time `json:"timeUpdated" xorm:"updated"`           // 记录更新时间
+	SoftDeletedAt uint64    `json:"softDeletedAt" xorm:"deleted notnull"` // 记录删除时间(时间戳形式)
```

Comment:
what will happen if no `notnull` specified?

Reply:
`SoftDeletedAt` column is set to `notnull` when created  in the db.
In the `xorm`, if it's `deleted` and not set to `notnull`, it will generate a sql as `insert into table(a,soft_deleted_at) VALUES(1,null)`, which will insert error and not satified with the `notnull` limit of the column in db.
![image](https://github.com/erda-project/erda/assets/86157286/9e46c600-e9a3-4972-8444-f0e419231d2b)